### ISSUE TITLE
A ScrollViewer with 0 width of height cannot be loaded

### ISSUE THREAD

**Issue by ramezgerges** [unoplatform/uno#13907]
### Current behavior

If you create a ScrollViewer with 0 width or height, it will never be loaded.

### Expected behavior

On WinUI, it loads normally, and it takes space. E.g. if the width is zero it will still get its height in e.g. a StackPanel. Moreover, if you tab until you focus whatever element is inside the ScrollViewer, you can scroll with the arrowkeys and the <Horizontal|Vertical>Offsets will reflect the keyboard movements like it normally would with non-zero dimensions.

### How to reproduce it (as minimally and precisely as possible)

```xaml
	<Border Width="0" Height="0">
		<ScrollViewer x:Name="sv" VerticalScrollMode="Enabled" HorizontalScrollMode="Enabled">
			<ItemsControl ItemsSource="{Binding VariableLengthItems}" />
		</ScrollViewer>
	</Border>
```

### Works on UWP/WinUI

None

**Comment by jeromelaban** [comment]
@dr1rrb does this look familiar?

**Comment by dr1rrb** [comment]
> @dr1rrb does this look familiar?

No, no idea. Most probably just a check somewhere to avoid to do useless work if 0,0 ...

**Comment by MartinZikmund** [comment]
Relevant

**Comment by MartinZikmund** [comment]
# Repro generator

Runtime test added in branch [`dev/mazi/repro/13907`](https://github.com/unoplatform/uno/compare/dev/mazi/repro/13907):
- `src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ScrollViewer.ZeroSize_13907.cs` → `Given_ScrollViewer_ZeroSize_13907.When_ScrollViewer_In_ZeroSize_Border_Should_Load_13907`

Mirrors the reported XAML: `Border` with `Width=0 / Height=0` containing a `ScrollViewer` with scroll modes enabled and a `TextBlock` content. Fails on Skia Desktop:
```
Timed out waiting for condition to be met. Timeout waiting on Microsoft.UI.Xaml.Controls.Border to be loaded
```
Even the outer 0×0 `Border` never raises `Loaded` on Uno (whereas WinUI loads it normally), confirming the issue.
